metricbeat/module/mssql/db/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"mssql",
+        "name":"db",
+        "rtt":44269
+    },
+    "mssql":{
+        "db":{
+            "example": "db"
+        }
+    },
+    "type":"metricsets"
+}
```

Comment:
Take a look to the `TestData` functions in the integration tests of other modules to see how this file is generated.